app/models/child.rb

```diff
@@ -83,12 +91,28 @@ def illinois_approval_amounts
   def nebraska_family_fee(filter_date)
     # "feature flag" for using live algorithms rather than uploaded data
     if Rails.application.config.ff_ne_live_algorithms
-      format('%.2f', active_nebraska_approval_amount(filter_date)&.family_fee)
+      family_fee = active_nebraska_approval_amount(filter_date)&.family_fee
+      family_fee || 0.00
     else
-      format('%.2f', temporary_nebraska_dashboard_case&.family_fee.to_f)
+      temporary_nebraska_dashboard_case&.family_fee.to_f
     end
   end
 
+  # NE dashboard family_fee calculator
+  def nebraska_earned_revenue(filter_date)
+    # feature flag for using live algorithms rather than uploaded data
+    if Rails.application.config.ff_ne_live_algorithms
+      revenue_less_family_fee(filter_date)
+    else
+      temporary_nebraska_dashboard_case&.earned_revenue&.to_f || 0.0
+    end
+  end
+
+  def revenue_less_family_fee(filter_date)
+    revenue = attendances.for_month(filter_date).pluck(:earned_revenue).sum - (active_nebraska_approval_amount(filter_date)&.family_fee || 0.0)
```

Comment:
Since the attendances know how much they've earned, we just sum it up and subtract the family fee.  If the result is negative, we return 0.